### DISCUSSION TITLE
Minio 必须要把ACCESS POLICY设置为PUBLIC才能正常使用吗？

### DISCUSSION THREAD

**Discussion by bestZwei** [lobehub/lobehub#4959] [General Question] [answered]
![image](https://github.com/user-attachments/assets/5a42ffef-eeb2-462c-92a0-51f9fc04786f)
当设置为私有时，我发现可以上传文件/图片，且存储桶存进去了这些内容，但是无法获取文件。
（其他环境是 Vercel+Clerk+Supabase Postgres）

![{5DFBC127-86AE-405C-B2A0-09E83D38AB02}](https://github.com/user-attachments/assets/8511c134-2c70-48c3-ab0a-a4163afcb7a1)

```
{
  "error": {
    "code": "count_token_messages_failed",
    "type": "new_api_error",
    "param": "",
    "message": "fail to get image from url: 403 Forbidden (request id: 20241210192023426354464aeTTFUF2)"
  },
  "endpoint": "https://api.***.org/v1",
  "provider": "openai"
}
```

**Comment by SpeedupMaster** [comment, marked as answer]
#3753 
S3_SET_ACL=0 试试这个
